app/controllers/api/conversion_hosts_controller.rb

```diff
@@ -1,5 +1,18 @@
 module Api
   class ConversionHostsController < BaseController
     include Subcollections::Tags
+
+    # Interface for the polymorphic resource, which could be either a Vm or
+    # a Host. Failure to provide a conversion host ID will raise an error.
+    #
+    # POST /api/conversion_hosts/1 {'action': 'resource'}
```

Comment:
is this a `post`?

Reply:
Yes. See my other comment. You can only do custom actions on a POST afaik, though perhaps @abellotti knows something I don't.